odin/assetlist.json

```diff
@@ -22,6 +22,27 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.svg",
 	"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.png"
       }
-    }
+    },
+    {
+        "description": "GEO token for ODIN Protocol",
+        "denom_units": [
+          {
+            "denom": "mGEO",
+            "exponent": 0
+          },
+          {
+            "denom": "GEO",
+            "exponent": 6
+          }
+        ],
+        "base": "mGEO",
+        "name": "GEO",
+        "display": "geo",
+        "symbol": "GEO",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/geo.svg",
```

Comment:
The svg file looks good, however, please move the images from the 'ODIN/images' folder and put them instead into the 'odin/images/' folder. Everything else in the 'ODIN/' folder should be removed, as there is already an 'odin' folder (all lowercase letters)